ansible/roles/k3s/files/start_k3s.yml

```diff
@@ -0,0 +1,42 @@
+- hosts: localhost
+  become: true
+  vars:
+    os_metadata: "{{ lookup('url', 'http://169.254.169.254/openstack/latest/meta_data.json') | from_json }}"
+    k3s_token: "{{ os_metadata.meta.k3s_token }}"
+    k3s_server_name: "{{ os_metadata.meta.k3s_server }}"
+    service_name: "{{ 'k3s-agent' if k3s_server_name is defined else 'k3s' }}"
+  tasks:
+    - name: Add the token for joining the cluster to the environment
+      no_log: false # avoid logging the server token
```

Comment:
I think `no_log: true` *prevents* logging stuff? can you check please.

Reply:
think I forgot to set this back after debugging